src/actions/audio.ts

```
@@ -51,6 +51,16 @@
   return getAudioPath(context, beat, audioFile);
 };
 
+export const listLocalizedAudioPaths = (context: MulmoStudioContext, targetLang?: string) => {
+  const lang = targetLang ?? context.lang;
+  return context.studio.script.beats.map((beat, index) => {
+    const multiLingual = context.multiLingual[index];
+    const text = localizedText(beat, multiLingual, lang);
+    console.log(text, multiLingual);
```

ESLint

[error] Unexpected console statement.

(no-console)